frontend/src/contexts/Providers.ts

```diff
@@ -0,0 +1,36 @@
+import Portis from '@portis/web3'
+import WalletConnectProvider from '@walletconnect/web3-provider'
+import Authereum from 'authereum'
+import Fortmatic from 'fortmatic'
+import Web3Modal from 'web3modal'
+
+const providerOptions = {
+  portis: {
+    package: Portis,
+    options: {
+      id: '8f5cf962-ad62-4861-ab0c-7b234b6e6cff'
```

Comment:
waaah. Don't put private secrets in git :( -> move them to .env and *after* the hackathon, rotate them on your account

Reply:
it's not my secret got it from 3idconnect repo.